tests/ttnn/unit_tests/module.mk

```diff
@@ -0,0 +1,29 @@
+
+
+TTNN_UNIT_TEST_DIRS := $(TT_METAL_HOME)/tests/ttnn/unit_tests/ $(TT_METAL_HOME)/tests/ttnn/unit_tests/operations/
+
+TTNN_UNIT_TESTS_SRCS := $(foreach dir,$(TTNN_UNIT_TEST_DIRS),$(wildcard $(dir)*.cpp))
+
+TTNN_UNIT_TESTS_INCLUDES := $(TT_METAL_INCLUDES) $(addprefix -I,$(TTNN_UNIT_TEST_DIRS))
+
+TTNN_UNIT_TESTS_LDFLAGS := -ltt_metal -ldl -lstdc++fs -pthread -lyaml-cpp -lgtest -lm
```

Comment:
Hmm, you'll probably need a `-lttnn -ltt_dnn` at the front here, assuming you're still calling your shared CPP library for `ttnn` `libttnn.so`.

Reply:
yes
updated to...
TTNN_UNIT_TESTS_INCLUDES := $(TEST_INCLUDES) $(TTNN_PYBIND11_INCLUDES)

TTNN_UNIT_TESTS_LDFLAGS := $(TTNN_PYBIND11_LDFLAGS) -lttnn -ldl -lstdc++fs -lgtest -lgtest_main -pthread -lm